src/views/templates/details/tabs/details/components/WorkloadProfile.tsx

```diff
@@ -0,0 +1,77 @@
+import * as React from 'react';
+import produce from 'immer';
+import useWorkloadProfile, {
+  workloadLabels,
+} from 'src/views/templates/list/hooks/useWorkloadProfile';
+
+import { ensurePath } from '@catalog/utils/WizardVMContext';
+import { TemplateModel, V1Template } from '@kubevirt-ui/kubevirt-api/console';
+import { useModal } from '@kubevirt-utils/components/ModalProvider/ModalProvider';
+import { useKubevirtTranslation } from '@kubevirt-utils/hooks/useKubevirtTranslation';
+import { VM_WORKLOAD_ANNOTATION } from '@kubevirt-utils/resources/vm';
+import { k8sUpdate } from '@openshift-console/dynamic-plugin-sdk';
+import {
+  Button,
+  DescriptionListDescription,
+  DescriptionListGroup,
+  DescriptionListTerm,
+} from '@patternfly/react-core';
+import { PencilAltIcon } from '@patternfly/react-icons';
+
+import { TemplateDetailsGridProps } from '../TemplateDetailsPage';
+
+import WorkloadProfileModal from './WorkloadProfileModal';
+
+const WorkloadProfile: React.FC<TemplateDetailsGridProps> = ({ template, editable }) => {
```

Comment:
can you add a WorkloadProfile type, e.g. `type WorkloadProfileNames = 'Desktop' | 'Server' | 'High performance'`; ?

Reply:
Done!